src/mzData.js

```diff
@@ -0,0 +1,69 @@
+import {parser} from 'sax';
+import {decoder, mergeSeries} from './utils';
+
+const xml = parser(true, {trim: true});
```

Comment:
it seems wrong to create the parser outside of the function. is it documented to do so?